src/scenario.h

```diff
@@ -302,7 +302,7 @@ typedef struct {
 	char custom_strings[0x8000];
 	uint32 game_ticks_1;
 	rct_ride rides[255];
-	uint16 word_01388698;
+	uint16 word_01388698; // TODO: rename to match RCT2_ADDRESS_SAVED_AGE ???
```

Comment:
I might have found the issue. Shouldn't [peep/peep.c:7002](https://github.com/OpenRCT2/OpenRCT2/blob/57420cbc669a1fa0f20088d42331b99c5beb09ee/src/peep/peep.c#L7002) actually be `mapElement->properties.path.ride_index`?